### PR TITLE
fix the 404 page and add redirects for blogs 

### PR DESCRIPTION
**Pull request by Tanuj22**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/404/404.style.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const L404SectionWrapper = styled.section`
 
     h1.message {
         font-size: 2rem;
-        line-height: 1.5rem;
+        line-height: 2rem;
     }
     h3.subtitle {      
         font-family: "Qanelas Soft Ultralight";
```
Comment (Nikhil-Ladha) [inline src/sections/404/404.style.js:14]: Any specific reason for this font?
If there is, please also add the generic family along with the font you are defining and for us that will be `sans-serif`. So, the CSS would be `font-family: 'Qanelas Soft Ultralight', sans-serif;`

File: src/sections/404/404.style.js [hunk 2]
```diff
@@ -17,9 +17,22 @@ const L404SectionWrapper = styled.section`
...
-    img.mesh {
-        margin-top: 50px
-        max-width: 40%;
+    .button-row{
+        display: flex;
+        flex-direction: column;
+        img{
+            display:block;
+            margin:auto;
+            margin-top: 50px
```
Comment (Nikhil-Ladha) [inline src/sections/404/404.style.js:26]: 'rem' please...

File: src/sections/404/404.style.js (hunk 1 above)
Comment (Tanuj22) [reply src/sections/404/404.style.js:14]: This font was used by Lee, so I didn't change it. Adding `sans-serif` to it.

Comment (Tanuj22) [reply src/sections/404/404.style.js]: Removed it. Instead changed the font-weight

Comment (Nikhil-Ladha) [reply src/sections/404/404.style.js]: Better!

**Comment by Nikhil-Ladha** [comment]
Looks good.
Just one thing, can you please revert your changes in the footer files (index, style) as I am working on it, and have made some changes, there will be unwanted conflicts 😆 
Also, we had redirects in blogs earlier right? Why add it to the MDX file too?

**Comment by Tanuj22** [comment]
> Looks good.
> Just one thing, can you please revert your changes in the footer files (index, style) as I am working on it, and have made some changes, there will be unwanted conflicts 
> Also, we had redirects in blogs earlier right? Why add it to the MDX file too?

The redirects for blogs weren't added before

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**